pkg/acceptance/c_test.go

```diff
@@ -194,55 +327,173 @@ int main(int argc, char const *argv[]) {
 				return 1;
 			}
 
 			PGint8 recvi8;
-			if (!PQgetf(result, 0, "%int8", 5, &recvi8)) {
-				fprintf(stderr, "ERROR: %s\n", PQgeterror());
+			if (!PQgetf(result, 0, "%int8", 6, &recvi8)) {
+				fprintf(stderr, "ERROR resultFormat=%d PQgetf(int8): %s\n", resultFormat, PQgeterror());
 				return 1;
 			}
 			if (recvi8 != i8) {
-				fprintf(stderr, "expected: %lld, got: %lld\n", i8, recvi8);
+				fprintf(stderr, "resultFormat=%d expected: %lld, got: %lld\n", resultFormat, i8, recvi8);
 				return 1;
 			}
 
+			// TODO(tamird,nvanbenschoten): implement interval binary encoding/decoding.
+			if (0) {
+				PGinterval recvinterval;
+				if (!PQgetf(result, 0, "%interval", 7, &recvinterval)) {
+					fprintf(stderr, "ERROR resultFormat=%d PQgetf(interval): %s\n", resultFormat, PQgeterror());
+					return 1;
+				}
+				if (!intervalEqual(recvinterval, interval)) {
+					fprintf(stderr, "resultFormat=%d expected:\n", resultFormat);
+					intervalPrint(interval);
+					fprintf(stderr, "got:\n");
+					intervalPrint(recvinterval);
+					return 1;
+				}
+			}
+
 			PGtext recvtext;
-			if (!PQgetf(result, 0, "%text", 6, &recvtext)) {
-				fprintf(stderr, "ERROR: %s\n", PQgeterror());
+			if (!PQgetf(result, 0, "%text", 7, &recvtext)) {
+				fprintf(stderr, "ERROR resultFormat=%d PQgetf(text): %s\n", resultFormat, PQgeterror());
 				return 1;
 			}
 			if (strcmp(recvtext, text)) {
-				fprintf(stderr, "expected: %s, got: %s\n", text, recvtext);
+				fprintf(stderr, "resultFormat=%d expected: %s, got: %s\n", resultFormat, text, recvtext);
 				return 1;
 			}
+
+			PGtimestamp recvts;
+			if (!PQgetf(result, 0, "%timestamp", 8, &recvts)) {
+				fprintf(stderr, "ERROR resultFormat=%d PQgetf(timestamp): %s\n", resultFormat, PQgeterror());
+				return 1;
+			}
+			// TODO(tamird,nvanbenschoten): fix ts decoding
+			// resultFormat={0,1} expected:
+			// epoch=0
+			// date:
+			// isbc=0
+			// year=2000
+			// mon=0
+			// mday=19
+			// jday=0
+			// yday=0
+			// wday=1
+			// time:
+			// hour=10
+			// min=41
+			// sec=6
+			// usec=0
+			// withtz=0
+			// isdst=24
+			// gmtoff=0
+			// tzabbr=-10321332
+			// got:
+			// epoch=1848018046
+			// date:
+			// isbc=0
+			// year=150924
+			// mon=3
+			// mday=3
+			// jday=56845012
+			// yday=93
+			// wday=1
+			// time:
+			// hour=1
+			// min=44
+			// sec=30
+			// usec=803968
+			// withtz=0
+			// isdst=-1
+			// gmtoff=0
+			// tzabbr=-10321332
+			if (0) {
+				if (!timestampEqual(recvts, ts)) {
+					fprintf(stderr, "resultFormat=%d expected:\n", resultFormat);
+					timestampPrint(ts);
+					fprintf(stderr, "got:\n");
+					timestampPrint(recvts);
+					return 1;
+				}
+			}
+
+			// TODO(dt): ERROR resultFormat={0,1} PQgetf(timestamp): Trying to get type 1114 'pg_catalog.timestamp' but server returned 1184 (position 1)
+			if (0) {
+				PGtimestamp recvtstz;
+				if (!PQgetf(result, 0, "%timestamp", 9, &recvtstz)) {
```

Comment:
Any chance the oid issue is because of `PQgetf(result, 0, "%timestamp", 9, &recvtstz)` instead of `PQgetf(result, 0, "%timestamptz", 9, &recvtstz)`? Just a guess